chia/data_layer/data_layer_server.py

```diff
@@ -110,7 +96,12 @@ async def file_handler(self, request: web.Request) -> web.Response:
         )
         return response
 
-    def _accept_signal(self, signal_number: int, stack_frame: Any = None) -> None:
+    def _accept_signal(
+        self,
+        signal_number: int,
+        stack_frame: Optional[FrameType],
+        loop: asyncio.AbstractEventLoop,
+    ) -> None:
         self.log.info("Got SIGINT or SIGTERM signal - stopping")
```

Comment:
it might even be nice to do something like: `f"Got signal {signal_number} - stopping"`